### ISSUE TITLE
tonal library missing

### ISSUE THREAD

**Issue by evavdborn** [gabelev/machine_music#3]
I cannot import the tonal library anymore.. Is it possible that it is removed?

**Comment by rudrathegreat** [comment]
I think so

**Comment by rudrathegreat** [comment]
Yes. I have searched everywhere for it. The main results I am finding are users on github called Tonal and even them don't have repos called tonal, let alone something to do with music in python. I'm sorry.
